package.json

```diff
@@ -1,21 +1,49 @@
 {
-    "name": "ronadatabase",
-    "version": "1.0.0",
-    "description": "",
+    "title": "Landing Page",
+    "name": "startbootstrap-landing-page",
+    "version": "5.0.8",
     "scripts": {
-        "start": "node server/app.js",
-        "local": "nodemon server/app.js"
+        "start": "node_modules/.bin/gulp watch"
+    },
+    "description": "An HTML landing page template built with Bootstrap",
+    "keywords": [
+        "css",
+        "sass",
+        "html",
+        "responsive",
+        "theme",
+        "template"
+    ],
+    "homepage": "https://startbootstrap.com/themes/landing-page",
+    "bugs": {
+        "url": "https://github.com/StartBootstrap/startbootstrap-landing-page/issues",
+        "email": "feedback@startbootstrap.com"
+    },
+    "license": "MIT",
+    "author": "Start Bootstrap",
+    "contributors": [
+        "David Miller (http://davidmiller.io/)"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/StartBootstrap/startbootstrap-landing-page.git"
```

Comment:
needs update

Reply:
This is from the Bootstrap template that I used for the landing page.